tests/Integration/Database/EloquentModelStringCastingTest.php

```diff
@@ -65,15 +65,15 @@ public function testSavingCastedAttributesToDatabase()
             'json_attributes' => ['json_key'=>'json_value'],
             'object_attributes' => ['json_key'=>'json_value'],
         ]);
-        $this->assertSame('{"key1":"value1"}', $model->getOriginal('array_attributes'));
+        $this->assertSame(['key1'=>'value1'], $model->getOriginal('array_attributes'));
         $this->assertSame(['key1'=>'value1'], $model->getAttribute('array_attributes'));
```

Comment:
Why did you change these tests? Aren't they probably protecting against some regression?

Reply:
I changed those because the test reflect the current situation, but not the desired situation.
If the model is instructed to cast a property to an array, i expect the output of getOriginal to be the same array, not a string.